### ISSUE TITLE
Fonts do not show on nerves deployment

### ISSUE THREAD

**Issue by argl** [boydm/scenic#104]
The hashing scheme described in the docs does evaluate the paths at compile time, which is usually the development machine’s paths and not the deployed (nerves) one. That leads to a problem where fonts do no show on nerves devices, since the resource path is different.

I “fixed” it for now by differentiating a bit more and hardcoding the nerves path (bad!):

```
  @font_hash_path :code.priv_dir(:my_app) |> Path.join("/static/fonts/myfont.otf")

  if @target == "host" do
    @font_path @font_hash_path
  else
    @font_path "/srv/erlang/lib/my_app-0.1.0/priv/static/fonts/myfont.otf"
  end

  @font_hash Scenic.Cache.Hash.file!(@font_hash_path, :sha)
```

and in `init/2`

```
def init(_, opts) do
  Scenic.Cache.File.load(@font_path, @font_hash)
...
```

I guess thats also the problem with the built-in fonts not showing in nerves. Probably a documentation issue first and foremost.

**Comment by boydm** [comment]
Just saw this too. Will be digging into it over the next day or so. Not sure what is going wrong yet.

**Comment by boydm** [comment]
Fixed in scenic_nerves_driver_rpi 0.9.2
Available on hex now.
